src/frontend/src/pages/Index/Settings/AdminCenter/HomePanel.tsx

```diff
@@ -73,9 +67,9 @@ export default function HomePanel(): JSX.Element {
         </Alert>
       )}
       <QuickAction ml='' />
-      <Title order={5}>
-        <Trans>System status</Trans>
-      </Title>
+      <StylishText size='lg'>
```

Comment:
I find this heavy use of accent colors fatiguing to the eye compared to a normal bold header